### PR TITLE
Feature/lazy load directive

### PR DESCRIPTION
**Pull request by strausr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +81,7 @@
     "rxjs": "^5.5.2",
     "systemjs": "^0.20.19",
     "tslint": "^5.8.0",
-    "typescript": "^3.1.6",
+    "typescript": "^2.9.2",
```
Comment (asisayag2) [inline package.json]: @strausr , why did we decrease version?

File: yarn.lock [hunk 2]
```diff
@@ -2,24 +2,64 @@
 # yarn lockfile v1
```
Comment (asisayag2) [inline yarn.lock]: Why do we need this file?

File: package.json (hunk 1 above)
Comment (strausr) [reply package.json]: This was the original version, the one that is compatible with our Angular version

File: yarn.lock (hunk 2 above)
Comment (strausr) [reply yarn.lock]: We don't. I just deleted it.

**Pull request merged**